TweetBook/Program.cs

```diff
@@ -2,27 +2,15 @@
 global using TweetBook.Services;
 global using Microsoft.EntityFrameworkCore;
 using TweetBook.Options;
-using Microsoft.AspNetCore.Authentication.JwtBearer;
-using Microsoft.IdentityModel.Tokens;
-using System.Text;
-using Microsoft.OpenApi.Models;
 using Microsoft.AspNetCore.Identity;
-using TweetBook.Authorization;
-using Microsoft.AspNetCore.Authorization;
-using TweetBook.Domain;
 using FluentValidation.AspNetCore;
-using System.Reflection;
 using TweetBook.Filters;
 using Swashbuckle.AspNetCore.Filters;
+using TweetBook.Installers;
 
 var builder = WebApplication.CreateBuilder(args);
```

Comment:
I don't like the fact that these lines are not inside of a method(since it's bad for readability), please arrange this file such that you are not leaving many lines that are external to a method.